PayPalUI/src/main/java/com/paypal/android/ui/paymentbutton/PayLaterButton.kt

```diff
@@ -0,0 +1,52 @@
+package com.paypal.android.ui.paymentbutton
+
+import android.content.Context
+import android.os.Build
+import android.util.AttributeSet
+import androidx.annotation.RequiresApi
+import com.paypal.android.ui.R
+
+
+/**
+ * PayLaterButton provides a PayPal PayLater button with the ability to modify the [color], [label], [shape],
```

Comment:
Can you modify the label on this button or no? The docs below make it seem like this can't be changed